src/main/resources/application-ootb.properties

```diff
@@ -0,0 +1,17 @@
+# =========================================================================
+# Spring-related.
+spring.config.activate.on-profile=ootb
+spring.config.import=optional:file:${user.home}/.${user.name}-conf/uh-groupings-ui-overrides.properties
+app.api.handshake.enabled=false
+
+# There are two options: GROUPER, OOTB : "GROUPER" is a real grouper api service while "OOTB" is a ootb grouper api service.
+# You can change active profiles from the RUN/DEBUG Configurations.
```

Comment:
The comment here is not very clear to me about changing active profiles. I'm assuming this is about adding the flag in `./mvnw clean spring-boot:run` to the environment to be dev, test, prod. etc. Maybe this can be removed since the API side does not have this comment.

Reply:
I'm going to add this feature in the api project default user related ticket too.

You can change the propfile like this below.

![20240510_110119](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/96459232/553e74aa-6bd4-4721-8543-2fb73e13808b)
![20240510_110139](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/96459232/503ecac9-9e8f-49cb-b0df-a289f99ef8c3)

Reply:
and then if you do `./mvnw clean spring-boot:run` it's going to use active file that we set